addons/account/models/account_move.py

```diff
@@ -718,27 +718,23 @@ def _compute_hide_post_button(self):
     def _compute_company_id(self):
         for move in self:
             if move.journal_id.company_id not in move.company_id.parent_ids:
-                move.company_id = (move.journal_id.company_id or self.env.company)._accessible_branches()[:1]
+                move.company_id = move.statement_line_id.company_id = move.payment_id.company_id = (move.journal_id.company_id or self.env.company)._accessible_branches()[:1]
```

Comment:
You can't assign a value like that outside of the compute's scope :sweat_smile: 
You already defined `company_id` as a computed field on statement_line/payment so you don't need this diff

Reply:
Looks like I found a way to not do that